### PR TITLE
Add AdvancedBan

### PR DESCRIPTION
**Pull request by Dsngxddd**
AdvancedBan support add

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -7,23 +7,26 @@ plugins {
...
+            url = "https://oss.sonatype.org/content/groups/public/"
+        }
+        maven { url "https://storehouse.okaeri.eu/repository/maven-public/" }
+        maven { url 'https://repo.triumphteam.dev/snapshots/' }
+        maven { url 'https://jitpack.io' }
+        maven { url 'https://repo.xenondevs.xyz/releases' }
+        maven { url 'https://repo.extendedclip.com/content/repositories/placeholderapi/' }
+        maven { url 'https://repo.essentialsx.net/releases/' }
+        mavenCentral()
     }
```
Comment (efekurbann) [inline build.gradle]: Could you please revert these changes? It's not in the right format for this file.
You just need to format the spaces correctly, other than that, looks good to me.

**Review by efekurbann — CHANGES_REQUESTED** [review]

**Comment by Dsngxddd** [comment]
fixed

**Review by efekurbann — APPROVED** [review]

**Comment by efekurbann** [comment]
Thank you for your contribution!
